### ISSUE TITLE
Unauthorised to upload content

### ISSUE THREAD

**Issue by PKISLOFF** [sapmentors/lemonaid#84]
Is this forthcoming at a later date when content page available?
At the moment from Profile, Media & Content tab, + button, select pdf file

> An error occurred while uploading the file
> <?xml version='1.0' encoding='UTF-8'?><error xmlns="http://schemas.microsoft.com/ado/2007/08/dataservices/metadata"><code/><message xml:lang="en-US">Unauthorized</message></error> (401)
> Close

**Comment by gregorwolf** [comment]
Yes, I think this functionality is not yet implemented. If you have some time you could just hide the area in the UI if you want.
